src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -47,6 +55,38 @@ export default class MyProfile extends Component {
     this.getProfileImage = this.getProfileImage.bind(this);
   }
 
+  componentDidMount() {
+    const { modules } = this.props;
+    const navLinks = modules.settings.map(({ links }) => links);
```

Comment:
To include entries from both app and settings-style modules, use something like
```
    const navLinks = ([].concat(modules.app, modules.settings)).map(({ links }) => links);
```

Reply:
thanks, updated the code